app/assets/stylesheets/variables/bootstrap.sass

```diff
@@ -95,14 +96,18 @@ $headings-font-weight:    500
 //
 //## Define common padding and border radius sizes and more. Values based on 14px text and 1.428 line-height (~20px to start).
 
-// $padding-base-vertical:     6px
-// $padding-base-horizontal:   12px
+$padding-base-vertical:     6px
```

Comment:
@stedmanblake you shouldn't need to define these. For all the commented out settings, they will remain those values by default. We'd only need to define items here that we plan to override (and you can still use the named variables as those come with importing bootstrap in the main app.css file that we do).